tools/cli/test/data/split/dev/openapi-mms.json

```diff
@@ -26028,9 +26028,6 @@
             "DigestAuth": []
           }
         ],
-        "x-xgen-changelog": {
```

Comment:
cleaning up x-xgen-changelog in most tests, but adding it to one of the endpoints